docs/extensions/advanced/storage-repositories.md

```diff
@@ -52,3 +52,173 @@ class KoalaCatcherExtension extends SimpleExtension
 ```
 
 **Note:** The above example uses "class name resolution" via the `::class` keyword.
+
+
+Entity / Repository and Alias.
+-----------------------------
+
+The below code is mapping a custom entity to a custom repository with the "gumtree" alias.
+```php
+protected function registerRepositoryMappings()
+{
+    return [
+        'gumtree' => [Entity\GumTree::class => Repository\GumTree::class],
+    ];
+}
+```
+That alias should match the **"unprefixed"** table name your entity data should be stored in. 
+
+If your data are going to be stored in the `bolt_foo` table, you alias should be `foo`
+
+**Note:** the default table prefix is `bolt_`
+
+Custom Repository to Manage Entity Defined in Contenttype.Yml
+-------------------------------------------------------------
+Sometimes it could be convenient to define content structure through the [contenttype.yml](../../contenttypes/intro)  file ( in particular to enjoy easy field declaration and to take advantage of the backend auto generated Create/Read/Update/Delete  UI ). If you also want to manage those content entities through a custom repository: we got you covered!
+
+Firstly You must create a custom Entity file which represent your content type and which extends `\Bolt\Storage\Entity\Content` and map that entity to your custom repository. Your custom repository must also extend `\Bolt\Storage\Repository\ContentRepository` and override the `createQueryBuilder` the whole is covered below:
```

Comment:
change to:

Firstly You must create a custom Entity file which represents your content type and which extends `\Bolt\Storage\Entity\Content` and then map that entity to your custom repository. Your custom repository must also extend `\Bolt\Storage\Repository\ContentRepository` and override the `createQueryBuilder` method as outlined below: